styles.css

```diff
@@ -0,0 +1,593 @@
+  float: right;
+}
+
+.right-elem a {
+  margin-right: 10px;
+  font-size: 12px;
+  text-decoration: none;
+  color: black;
+}
+
+.icon-arrow i {
+  color: black;
+}
+
+.clas {
+  display: inline-block;
+}
+
+/* MAIN SECTION STYLING */
+.out-container {
+  margin-top: 50px;
+  margin-bottom: 50px;
+  display: flex;
+  justify-content: center;
+}
+
+.socialiAnchors {
+  color: black;
+}
+
+.container {
+  width: 956px;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  font-family: 'Merriweather', serif;
+}
+
+.main-text {
+  font-size: 18px;
+  margin: 10px 150px;
+  line-height: 30px;
+}
+
+.container h2 {
+  margin: 30px 150px;
+  font-family: 'Balsamiq Sans', cursive;
+  font-size: 35px;
+}
+
+.image1 {
+  text-align: center;
+}
+
+.image1 img {
+  width: 956px;
+  height: auto;
+}
+
+.img1desc {
+  font-size: 12px;
+  text-align: left;
+  display: block;
+}
+
+.imgpar {
+  display: block;
+  text-align: left;
+  font-size: 10px;
+  color: grey;
+}
+
+/* AUTHOR SOCIAL CONTACT */
+.authorName {
+  font-size: 14px;
+  font-weight: bold;
+  text-decoration: none;
+  color: black;
+}
+
+.grid {
+  display: grid;
+  width: 656px;
+  margin-left: 150px;
+  margin-top: 15px;
+}
+
+.date-container {
+  align-items: center;
+}
+
+.date {
+  font-size: 12px;
+  grid-column: 1 / 2;
+}
+
+.social {
+  grid-column: 4 / 6;
+  display: flex;
+  justify-content: flex-end;
+}
+
+.social li {
+  display: inline-flex;
+}
+
+.social-media {
+  height: 40px;
+  width: 40px;
+  display: flex;
+  margin-left: 10px;
+  border: #eee 1px solid;
+  border-radius: 50%;
+  background-color: #eee;
+  align-items: center;
+  justify-content: center;
+  font-size: 12px;
+}
+
+.social a {
+  text-decoration: none;
+  color: black;
+}
+
+.social-media:hover {
+  background-color: white;
+}
+
+.social-media-two {
+  height: 40px;
+  width: 40px;
+  display: flex;
+  margin-left: 10px;
+  align-items: center;
+  justify-content: center;
+  font-size: 25px;
+}
+
+/* MID TEXT ADVERTISEMENT */
+.text-hr {
+  color: white;
+  width: 50px;
+  border-top: 1px solid #ccc;
+  margin-left: 453px;
+  margin-top: 10px;
+  margin-bottom: 10px;
+}
+
+.advertisingConta {
+  text-align: center;
+  font-family: 'Rubik', sans-serif;
+  font-weight: bold;
+  display: flex;
+  flex-direction: column;
+  font-size: 12px;
+  padding: 10px;
+}
+
+.advertiPar {
+  margin-bottom: 5px;
+}
+
+.adverAnchor {
+  text-decoration: none;
+  color: #326891;
+}
+
+.textAnchor {
+  text-decoration: underline;
+  color: #326891;
+}
+
+/* ASIDE BOX STYLING */
+.asideBox {
+  margin-top: 1570px;
+  width: 250px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  position: absolute;
+  right: 50px;
+}
+
+.asideCon {
+  margin: 10px 0;
+}
+
+.floatAnchors {
+  text-decoration: none;
+  color: black;
+}
+
+.floatAnchors2 {
+  text-decoration: none;
+  color: black;
+}
+
+.floatAnchors2:hover {
+  color: grey;
+}
+
+.floatAnchors:hover {
+  text-decoration: underline;
+}
```

Comment:
This selector has the same styles as the selector on line 591. Group selectors to minimize code.